src/emcpy/plots/create_map.py

```diff
@@ -0,0 +1,294 @@
+        """
+
+        self.ax.set_xlabel(xlabel=xlabel, loc=loc, fontsize=fontsize,
+                           fontweight=fontweight, color=color)
+
+    def add_ylabel(self, ylabel,
+                   loc='center',
+                   fontsize=12,
+                   fontweight='normal',
+                   color='k'):
+        """
+        Plots y label to map axes.
+
+        Args:
+            ylabel : (str) Text to use for y label
+            loc : (str; default='center') Location of title
+            fontsize : (int; default=12) Text font size
+            fontweight : (str; default='normal') Text font weight
+            color : (str; default='k') Text font color
+        """
+
+        self.ax.set_ylabel(ylabel=ylabel, loc=loc, fontsize=fontsize,
+                           fontweight=fontweight, color=color)
+
+    def add_colorbar(self, label=None, label_fontsize=12, extend='neither'):
+        """
+        Creates new axes and plots colorbar to figure.
+
+        Args:
+            label : (str; default=None) Colorbar label
+            label_fontsize : (int; default=12) Colorbar label font size
+            extend : (str; default='neither') Extends min/max side of colorbar
+        """
+        if 'cs' in dir(self):
+            cax = self.fig.add_axes([self.ax.get_position().x1 + 0.02,
+                                     self.ax.get_position().y0, 0.025,
+                                     self.ax.get_position().height])
+
+            cb = plt.colorbar(self.cs, extend=extend, cax=cax)
+            cb.set_label(label, fontsize=label_fontsize)
+
+        else:
+            raise TypeError('Data object has colorbar set to False.' +
+                            'Set obj.colorbar=True')
+
+    def add_grid(self,
+                 linewidth=1,
+                 color='gray',
+                 alpha=None,
+                 linestyle='-'):
+        """
+        Plots gridlines on map axes.
+
+        Args:
+            linewidth : (int; default=1) Line thickness
+            color : (str; default='gray') Line color
+            alpha : (float; default=None) The alpha value,
+                    between 0 (transparent) and 1 (opaque)
+            linestyle : (str; default='-') Line style
+        """
+        self.ax.gridlines(crs=self.projection.projection, linewidth=linewidth,
+                          color=color, alpha=alpha, linestyle=linestyle)
+
+    def return_figure(self):
+        """
+        Returns the figure created.
+        """
+
+        return self.fig
+
+    def _add_coastlines(self):
+        """
+        Add coastline to map axes. (Only feature that currently works)
+        """
+        self.ax.add_feature(cfeature.GSHHSFeature(scale='auto'))
+
+        # self.ax.add_feature(cfeature.COASTLINE)
+
+    def _add_borders(self):
+        """
+        Add country borders to map axes.
+        """
+        return
+        # self.ax.add_feature(cfeature.BORDERS)
```

Comment:
why are these commented out? Just because the shape files are missing on Hera? Will it not work with them included if we don't call these functions yet?

Reply:
If you do not call these functions it should be fine. However if you do, you get the following warning:
```
DownloadWarning: Downloading: https://naciscdn.org/naturalearth/50m/cultural/ne_50m_admin_0_boundary_lines_land.zip
  warnings.warn('Downloading: {}'.format(url), DownloadWarning)
```

and you get a timeout error and the figure never generates.